### PR TITLE
 Double back press to exit the app #110

### LINKED ISSUE #110: Add 'Press again to exit...'
Currently hitting the Back button once exits the Tweet Harvesting Activity. The user might end up hitting the Back button by mistake which would cause an unintentional exit from the app. Thus  it would be a better option to ask the user to hit the Back button twice to exit the app.

### PR DESCRIPTION
**Pull request by riyasingh1004**
The user needs to press the back button twice to exit the app.

Fix #110 

Changes: [Add here what changes were made in this issue and if possible provide links.]

Screenshots for the change:

### CODE REVIEW

**Review by championpaddler — COMMENTED** [review]
@riyasingh1004  Please fix failing travis.

**Comment by riyasingh1004** [comment]
Can you please help me understand why the Travis CI build is failing. When I am running it on my system its working properly.

File: app/src/main/java/org/loklak/wok/ui/activity/TweetHarvestingActivity.java [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
-import android.os.Bundle;
-import android.os.Handler;
-import android.support.annotation.Nullable;
-import android.support.v7.app.AppCompatActivity;
-import android.view.WindowManager;
-import android.widget.Toast;
-
-import org.loklak.wok.ui.fragment.TweetHarvestingFragment;
-import org.loklak.wok.R;
+import . . .
```
Comment (championpaddler) [inline app/src/main/java/org/loklak/wok/ui/activity/TweetHarvestingActivity.java]: @riyasingh1004  This is causing the error.

**Review by championpaddler — CHANGES_REQUESTED** [review]
@riyasingh1004

**Comment by riyasingh1004** [comment]
The Travis CI check is done but now codecov/patch is showing some error. Can you please explain about it.

**Comment by championpaddler** [comment]
> The Travis CI check is done but now codecov/patch is showing some error. Can you please explain about it.

@riyasingh1004  Os.handler package is removed , I suppose.

**Comment by riyasingh1004** [comment]
Should I add it?

**Comment by championpaddler** [comment]
> Should I add it?
Try .

**Comment by mariobehling** [comment]
Thank you.

**Pull request merged**